traffic_ops/app/db/admin.go

```diff
@@ -463,3 +564,49 @@ func main() {
 		die("invalid command: " + userCmd)
 	}
 }
+
+func initMigrate() {
+	var err error
+	connectionString := fmt.Sprintf("%s://%s:%s@%s:%s/%s?sslmode=%s", DBDriver, DBUser, DBPassword, HostIP, HostPort, DBName, SSLMode)
+	if !TrafficVault {
```

Comment:
this might be easier to read if we remove the negation and switch the branches
```
if TrafficVault { 
    // do TV stuff
} else {
    // do regular stuff
}
```

Reply:
Negation removed and branches switched in 42c3bb9a74.